jax/experimental/compilation_cache/file_system_cache.py

```diff
@@ -12,15 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Optional
 import os
+import tempfile
+from typing import Optional
 
 class FileSystemCache:
 
-  def __init__(self, path: str):
+  def __init__(self, path: str, max_cache_size_bytes = 32000000000):
```

Comment:
For some reason we don't put spaces around the `=` sign when using keyword argument values. See https://google.github.io/styleguide/pyguide.html#36-whitespace (paragraph starting with "Never use spaces around").